kimchi/src/prover.rs

```diff
@@ -191,6 +191,15 @@ where
         //~ 1. Setup the Fq-Sponge.
         let mut fq_sponge = EFqSponge::new(index.fq_sponge_params.clone());
 
+        //~ 1. Absorb the verifier index
+        let verifier_index = index.verifier_index();
```

Comment:
Doesn't need to be done now, but we should compute this once while we're creating the index and cache the initialised sponge in the index. We don't really want to recommit to everything in the verifier index in every proof.